src/main/resources/site/pages/default/default.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width"/>
+
+    <title>Superhero theme</title>
+
+    <!--[if lt IE 9]>
+    <script data-th-src="${assetUrls.html5Js}" src="../assets/js/html5.js" type="text/javascript"></script>
+    <![endif]-->
+
+    <link rel="stylesheet" id="style-css" data-th-href="${assetUrls.styleCss}" href="../assets/css/style.css" type="text/css" media="all"/>
+    <link rel="stylesheet" id="enonic-css" data-th-href="${assetUrls.enonicCss}"  href="../assets/css/enonic.css" type="text/css" media="all"/>
+    <link rel="stylesheet" data-th-href="${assetUrls.flexsliderCss}"  href="../assets/css/flexslider.css" type="text/css" media="all"/>
+    <link rel="stylesheet" id="superhero-carrois-gothic-css" data-th-href="${assetUrls.fontCarroisCss}" href="https://fonts.googleapis.com/css?family=Carrois+Gothic&amp;ver=4.1-alpha-30340" type="text/css" media="all"/>
+    <style type="text/css">.recentcomments a{display:inline !important;padding:0 !important;margin:0 !important;}</style>
+
+    <script data-th-src="${assetUrls.jqueryJs}" src="http://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script type="text/javascript" data-th-src="${assetUrls.superheroJs}" src="../assets/js/superhero.js"></script>
+    <script type="text/javascript" data-th-src="${assetUrls.flexsliderJs}" src="../assets/js/jquery.flexslider-min.js"></script>
+    <meta name="robots" content="noodp" />
+    <meta name="generator" content="Enonic XP" />
+    <meta name="author" content="Enonic" />
+
+    <style data-th-if="${backgroundImageUrl}"
+           data-th-utext="|body.custom-background { background-image: url('${backgroundImageUrl}'); background-repeat: repeat; background-position: top left; background-attachment: scroll;|"
```

Comment:
Can you just add this as normal text? 
```
<style data-th-if="${backgroundImageUrl}">
    body.custom-background { 
        .... 
    }
</style>
```

Have not seen an id on a style tag before.

Reply:
And for the url: 
`background-image: url('[[${backgroundImageUrl}]]')`
Might be this is not prettier, keep it as is if you feel like it.

Reply:
> Have not seen an id on a style tag before.

Me neiter. It's in multiple places here. Need to figure out if that has a use or is even valid.

Reply:
Fixed. The IDs weren't used anywhere else.